website/src/pages/Blog/Tag/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from "react";
+import useStyles from "./styles";
+import { useTranslation } from "react-i18next";
+import {
+  Container,
+  Grid
+} from "@material-ui/core";
+import Footer from "../../../components/Footer";
+import { METADATA_TYPES } from "../../../constants";
+import Sponsor from "../../../components/Sponsor";
+import Pagination from "@material-ui/lab/Pagination";
+import { pageCount } from "../../../utils/getPageCount";
+import SeoJson from "../../../resources/seo.json";
+import { useCurrentHost } from "../../../hooks/useCurrentHost";
+import { Metadata } from "../../../components/Metadata";
+import BlogCard from "../../../components/BlogCard";
+import { useTag } from "../../../hooks/extractBlogPath";
+import { useHistory } from "react-router-dom";
+
+interface blogObject { 
+  title: string;
+  author: string;
+  excerpt: string;
+  author_info: string;
+  date: string;
+  tags: Array<string>;
+  content: string;
+  id: number;
+  slug: string; 
+}
+
+const Tag: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const { currentOrigin, currentLocation } = useCurrentHost();
+  const [filteredData, setFilteredData] = useState<blogObject[]>([
+    { title: "",
+      author: "",
+      excerpt: "",
+      author_info: "",
+      date: "",
+      tags: [],
+      content: "",
+      id: 0,
+      slug: "" }
+  ]);
+  const selectedTag = useTag();
+  const itemsPerPage = 6;
+  const [page, setPage] = React.useState<number>(1);
+  const history = useHistory();
+
+  
+
+  const handleTagSelect = (tag: string) => {
+    if(selectedTag !== tag){
```

Comment:
the check should be something like this 

```suggestion
    if(selectedTag !== tag.toLowerCase().replace(/[^\w ]+/g,'').replace(/ +/g,'-')){
```